### DISCUSSION TITLE
Store scale/weight data for long time

### DISCUSSION THREAD

**Discussion by cold1ce** [dckiller51/bodymiscale#200] [General]
Is there an easy way to save only the sensor data of the scale for a longer time without changing the purge_keep_days setting of Home Assistant? (This would mean that ALL sensor data would be saved for a longer time, which I want to avoid). 

I would like to display a graph showing, for example, the weight history over a year. 

Does anyone have any good tips here or have solved that problem already?

**Comment by Bretthansenpc** [comment]
This is what I use for any data that I want to go back to. 

[Google sheets](https://www.home-assistant.io/integrations/google_sheets/)

I create an automation that sends any data I need to keep to a Google sheet and triggers when there's a new value. 

That way I can create graphs and such from the data.
